plugins/kettle5-log4j-plugin/ivy.xml

```diff
@@ -25,7 +25,9 @@
 
     <dependency org="log4j" name="log4j" rev="1.2.17" conf="default->default" transitive="false"/>
 
-    <dependency org="junit" name="junit" rev="4.7" conf="test->default"/> 
+    <!-- test dependencies -->
+    <dependency org="junit" name="junit" rev="4.11" conf="test->default"/> 
+    <dependency org="org.mockito" name="mockito-all" rev="1.9.5" conf="test->default" transitive="false" />
```

Comment:
@mattyb149  @akhayrutdinov said that Joe rejecting COMP for testing artifacts e.g. http://jira.pentaho.com/browse/COMP-1412,
http://jira.pentaho.com/browse/COMP-1413.